ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/netty/NettyP2PNetwork.java

```diff
@@ -166,18 +168,14 @@ public NettyP2PNetwork(
       final PeerRequirement peerRequirement,
       final PeerBlacklist peerBlacklist,
       final MetricsSystem metricsSystem,
-      final NodeWhitelistController nodeWhitelistController) {
+      final NodeWhitelistController nodeWhitelist) {
```

Comment:
Why the name change?  This object is still the controller, not that actual whitelist?